jcommune-view/jcommune-web-controller/src/test/java/org/jtalks/jcommune/web/controller/CodeReviewControllerTest.java

```diff
@@ -101,6 +104,24 @@ public void createPage() throws NotFoundException {
     }
 
     @Test
+    public void showNewCodeReviewPageShouldShowDraft() throws NotFoundException {
```

Comment:
We actually have something started in Component Tests for topic creation, maybe you could easily write this kind of test there. Take a look at `TopicControllerTest` and tell me what you think.